modules/lang-painless/src/main/java/org/elasticsearch/painless/action/PainlessExecuteAction.java

```diff
@@ -809,12 +810,18 @@ private static Response prepareRamIndex(
                     BytesReference document = request.contextSetup.document;
                     XContentType xContentType = request.contextSetup.xContentType;
                     String id;
```

Comment:
I think this variable is unused now. Maybe remove it?

Reply:
Done.